addons/find.lua

```diff
@@ -12,10 +12,11 @@
 
 local msg = require "mp.msg"
 local fb = require "file-browser"
-local input = require "user-input-module"
+local input_loaded, input = pcall(require, "mp.input")
+local user_input_loaded, user_input = pcall(require, "user-input-module")
 
 local find = {
-    version = "1.3.0"
+    version = "1.4.0"
```

Comment:
The version field is for the API version that the addon requires, not a version number for the addon itself (this is a little misleading I admit).

```suggestion
    version = "1.3.0"
```